_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-external-media.php

```diff
@@ -249,7 +249,7 @@ public function get_connection_details( \WP_REST_Request $request ) {
 		$service    = rawurlencode( $request->get_param( 'service' ) );
 		$wpcom_path = sprintf( '/meta/external-services/%s', $service );
 
-		$response = Client::wpcom_json_api_request_as_user( $wpcom_path, '2', array(), null, 'rest' );
+		$response = Client::wpcom_json_api_request_as_user( $wpcom_path, '1.1', array(), null, 'rest' );
```

Comment:
🤔 have we changed these values before?

Reply:
Possibly? In my testing just now `2` didn't give me a result, though, `1.1` did.

Reply:
Oh, I see it was [modified here](https://github.com/Automattic/jetpack/commit/906f2d43e575ea2d8e16c925aeb58e451b9d29bd#diff-6c5528991a5cda0f1a07c07e5e9fd9f3R249).